### PR TITLE
Add callout section to the Layer5 Discussion Forum

### PR DESCRIPTION
**Pull request by kayceeDev**
Signed-off-by: kayceeDev <Nwobodoe71@gmail.com>

**Description**

This PR fixes #2009 [Community and FAQ] Missing callout sections to the Layer5 Discussion Forum

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/pages/company/faq.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ import Footer from "../../sections/General/Footer";
 
 import { GlobalStyle } from "../../sections/app.style";
 import theme from "../../theme/app/themeStyles";
+import JoinCommunity from "../../sections/Community/Join-community";
+import Lee_workshop from "../../sections/Community/Lee_Workshop.png"
+
```
Comment (Chadha93) [inline src/pages/company/faq.js:15]: Could you please remove this unwanted line space

File: src/sections/Community/Join-community/index.js [hunk 2]
```diff
@@ -27,8 +29,16 @@ const JoinCommunityWrapper = styled.div`
...
-    }
-`;
+  
+      }
+
+      .callout{
+        display: grid;
+
+      }
+`
```
Comment (Chadha93) [inline src/sections/Community/Join-community/index.js:39]: Here also format the spacing

File: src/sections/Community/Join-community/index.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ const JoinCommunityWrapper = styled.div`
         background-size: cover;
         background-position: center;
         margin: auto;
+
```
Comment (Chadha93) [inline src/sections/Community/Join-community/index.js:14]: Here too

**Review by Chadha93 — CHANGES_REQUESTED** [review]
Also please link the issue number against the "#" in the PR description.

File: package.json [hunk 4]
```diff
@@ -74,6 +74,7 @@
     "react-twitter-embed": "^3.0.3",
     "react-vertical-timeline-component": "^3.3.0",
     "react-visibility-sensor": "^5.1.1",
+    "sharp": "^0.28.3",
```
Comment (Nikhil-Ladha) [inline package.json:77]: We don't need this new package. So, please revert this change and the changes to the package-loock.json file.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit fceb02ef517023c3e36802a30e9878f790f2e2ca at: https://60f69d5cc5ed53b100ec52af--layer5ng.netlify.app

File: package.json (hunk 4 above)
Comment (kayceeDev) [reply package.json:77]: without this package, it couldn't run on my local machine.

**Comment by leecalcote** [comment]
Thanks for doing this, @kayceeDev. We need people to be aware of the discussion forum. This helps.

**Review by kayceeDev — COMMENTED** [review]
Signed-off-by: kayceeDev <Nwobodoe71@gmail.com>

**Description**

This PR fixes #2009 [Community and FAQ] Missing callout sections to the Layer5 Discussion Forum and removes unwanted line spaces

**Notes for Reviewers**


**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

**Comment by kayceeDev** [comment]
> Thanks for doing this, @kayceeDev. We need people to be aware of the discussion forum. This helps.

😎

**Comment by l5io** [comment]
🚀 Preview for commit 4250e2083733bf4738bc850db32eacec397b7bb5 at: https://60f9db72d8a8f100d1ec6651--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
@leecalcote I am not sure if we want both the buttons to be present like this?
Screenshot from 2021-07-23 17-28-13

**Comment by kayceeDev** [comment]
@Nikhil-Ladha Can you elaborate why you prefer it having two sections rather than a section that talks about the two community which also  contains  two links that redirects anyone interested to join. I want to see from you perspective and learn.

**Comment by leecalcote** [comment]
@kayceeDev you might choose to "Resolve" any of the comments above that you consider you have addressed. This will help a bit in reviewing the state of the PR.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha Can you elaborate why you prefer it having two sections rather than a section that talks about the two community which also contains two buttons that redirects anyone interested to join. I want to see from you perspective and learn.

It seems crowded to me, not sure though 🤔

**Comment by leecalcote** [comment]
Oh, I'm just seeing the question and the screenshot. No, we don't want both of these actions in the same section. They are two independent calls to action to the visitor. Let's get a separate "advertisement" (a separate callout) to the discussion forum, please.

**Comment by Tanuj22** [comment]
Fixed in #2065
